system-parachains/asset-hubs/asset-hub-polkadot/tests/snowbridge.rs

```diff
@@ -57,9 +59,16 @@ fn network_export_table_works() {
 				None,
 			),
 			// From Ethereum with the Sepolia chain ID instead of Mainnet, not matched.
-			(NetworkId::Ethereum { chain_id: 11155111 }, Here, None),
+			(Ethereum { chain_id: 11155111 }, Here, None),
 		];
 
+		parameter_types! {
+			pub BridgeTable: Vec<NetworkExportTableItem> =
+				Vec::new().into_iter()
+				.chain(to_ethereum::EthereumBridgeTableV1::get())
+				.collect();
+		}
```

Comment:
why the need to override `BridgeTable` here?

does the "real" one not work for this test? why not?

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/796/commits/c89500c233116dc7c989ca7a1e981329a08df0c4.